open-company/open-company-web
# Expanded QP

Card: https://trello.com/c/CVQFv3G0

To test:
- do you see QP if you login as admin?
- do you see QP if you login as contributor?
- do you NOT see QP if you login as viewer?
- do you see QP if you login as viewer with at least one private section with Edit access?
- in all the above cases QP has an editable board selected by default?
- try adding a post from QP
- try adding a post from QP with abstract
- try adding a post from QP with other media (gif, video, picture, attachments etc...)
- add a post but refresh before you publish it
- post reopen in fullscreen?
- fullscreen still looks good?
